From 60518925be620adfbd523c16521b0a1ca8a1aaad Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz@archlinux.org>
Date: Tue, 19 Jan 2021 18:59:59 -0500
Subject: [PATCH] setup.cfg: remove build-time dependency on things unrelated
 to building

pep517 is a command-line tool used by developers interested in building
any package, it does not need to be listed as a requirement which
python -m pep517.build installs into the isolated build environment.

wheel is used optionally by setuptools to run bdist_wheel and is
required by the pep517 build backend... but this is only needed if
you're using pyproject.toml's build-system.requires, so this is a
duplicate line.

[patch regenerated against PyPI tarball with funky added whitespace]
---
 setup.cfg | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index d8436cc..615630d 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -41,8 +41,6 @@ install_requires =
 setup_requires = 
 	setuptools-scm>=3.5.0
 	setuptools>=42.0
-	wheel
-	pep517
 packages = py7zr
 
 [options.entry_points]
-- 
2.30.0

